### ISSUE TITLE
Take over compiler/machine/backend configuration from act

### ISSUE THREAD

**Issue by MattWindsor91** [MattWindsor91/act-tester#7]
I’m currently making the tester depend on `act` for all of the compiler/machine/backend introspection.  Once this tester is complete, it’ll likely be the only part of `act` depending on that information, in which case I wager `act-tester` would do a better job of maintaining that configuration itself.

This would then reduce `act` to a fuzzer and C litmus test manipulator, which seems closer to what it actually is.

However, this is likely to not be a priority this side of a paper, so I’m wishlisting it.  (Labelling it WONTFIX for now, but I’ll eventually make a new label.)

**Comment by MattWindsor91** [comment]
This has ended up becoming the case out of convenience when implementing other bits of the tester, so I'm closing this.
